pkg/cmd/api/api.go

```diff
@@ -134,6 +135,7 @@ original query accepts an '$endCursor: String' variable and that it fetches the
 	cmd.Flags().BoolVarP(&opts.ShowResponseHeaders, "include", "i", false, "Include HTTP response headers in the output")
 	cmd.Flags().BoolVar(&opts.Paginate, "paginate", false, "Make additional HTTP requests to fetch all pages of results")
 	cmd.Flags().StringVar(&opts.RequestInputFile, "input", "", "The file to use as body for the HTTP request")
+	cmd.Flags().BoolVar(&opts.Silent, "silent", false, "Silence the output")
```

Comment:
Let's try to be more descriptive here: how does “Do not print response body on output" sound? The goal would be to describe what "silence" means in this case, since the word could be ambiguous